pgx-macros/src/rewriter.rs

```diff
@@ -150,9 +140,9 @@ impl PgGuardRewriter {
                             arg_list.extend(quote! { #ident, });
                         }
                     } else {
-                        eyre::bail!(
+                        panic!(
```

Comment:
Ideally we'd use `syn::Error` here, since panic will give a pretty bad error message inside a macro/build script.